kube-enforcer/values.yaml

```diff
@@ -172,6 +172,9 @@ podAnnotations: {}
 podLabels: {}
 affinity: {}
 
+podDisruptionBudget:
+  minAvailable: 1
```

Comment:
For this initial cut I am not adding support for `maxUnavailable` but would be happy to include it if desired.